app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/ComponentNavigationActivity.java

```diff
@@ -151,7 +151,7 @@ protected void onCreate(Bundle savedInstanceState) {
   public void onMapReady(@NonNull MapboxMap mapboxMap) {
     mapboxMap.setStyle(new Style.Builder().fromUrl(getString(R.string.navigation_guidance_day)), style -> {
       mapState = MapState.INFO;
-      navigationMap = new NavigationMapboxMap(mapView, mapboxMap);
+      navigationMap = new NavigationMapboxMap(mapView, mapboxMap, null);
```

Comment:
Can we replace all this constructor calls (with 3-rd null param) with constructor without null parameter?

Reply:
done